src/client/state/Notifications.js

```diff
@@ -190,6 +231,14 @@ class Notifications extends EventEmitter {
         this.deleteNoti(room.roomId);
       }
     });
+
+    this.matrixClient.on('sync', (state) => {
+      if (state === 'SYNCING') {
+        this._isSyncing = true;
+      } else if (state === 'STOPPED' || state === 'ERROR') {
+        this._isSyncing = false;
+      }
+    });
```

Comment:
this code can be removed if `this.matrixClient.getSyncState()` is used at line: 213. I am not sure what else purpose it serve.